fred2/ignoreordersdlg.h

```diff
@@ -30,8 +30,8 @@ class ignore_orders_dlg : public CDialog
 	ignore_orders_dlg(CWnd* pParent = NULL);   // standard constructor
 
 	ship *m_shipp;
-	check_box_info check_boxes[MAX_CHECKBOXES];
-	int m_num_checks_active;
+	SCP_vector<size_t> m_orderList;
```

Comment:
Why `size_t`?

I would think that type would be used only to represent a "size" of some kind.

Reply:
Cause its the size of the vector.
Since all commands now represent indices of a vector, a size_t is appropriate. Also, it's already used as command in a fair few other places, so this is to keep consistent